src/components/newsletter-signup-modal/NewsletterSignupModal.module.scss

```diff
@@ -0,0 +1,118 @@
+@import '../../utils/variables';
+
+.container {
+  color: $white;
+
+  @media (min-width: $medium) {
+    padding: 8px;
+  }
+}
+
+.header {
+  font-weight: $blackBold;
+  font-size: 48px;
+  margin-bottom: 8px;
+
+  @media (min-width: $medium) {
+    font-size: 64px;
+  }
+}
+
+.subHeader {
+  font-size: 20px;
+
+  @media (min-width: $medium) {
+    font-size: 24px;
+  }
+}
```

Comment:
Similar thing with nesting here - eg is subHeader a child of the header (I haven't check so maybe not)?

Reply:
`header` and `subHeader` are `Text` components at the same level on the component.